docs: improve server-side docs (#134)
nuxt-modules/turnstile: README.md

   ```js
   export default defineNuxtConfig({
     modules: ['@nuxtjs/turnstile'],
     
     turnstile: {
       siteKey: '<your-site-key>',
     },
     
     runtimeConfig: {
       turnstile: {
         secretKey: process.env.NUXT_TURNSTILE_SECRET_KEY || '',
       },
     },
   })
   ```

   **Tip**: At runtime you can override site and secret keys with the `NUXT_TURNSTILE_SECRET_KEY` and `NUXT_PUBLIC_TURNSTILE_SITE_KEY` environment variables.

## Usage

> To verify, you need to
> - Include turnstile in your client (for example to build a contact form)
> - Verify the token, you get from the client, on the server (before sending the email out)

Reviewer: ```suggestion
To use Turnstile, you will likely want to:
- Use the `<Turnstile>` component in your app (for example to build a contact form)
- On your server when you are processing an API request or a form submission, you will want to verify the token (for example, before sending the email out)
```

> To use Turnstile, you will likely want to:
> - Use the `<Turnstile>` component in your app (for example to build a contact form)
> - On your server when you are processing an API request or a form submission, you will want to verify the token (for example, before sending the email out)

Reviewer: ```suggestion
To use Turnstile, you will likely want to:

- Use the `<Turnstile>` component in your app (for example to build a contact form)
- On your server when you are processing an API request or a form submission, you will want to verify the token (for example, before sending the email out)
```

   ```js
   export default defineNuxtConfig({
     modules: ['@nuxtjs/turnstile'],
     
     turnstile: {
       siteKey: '<your-site-key>',
     },
     
     runtimeConfig: {
       turnstile: {
         secretKey: process.env.NUXT_TURNSTILE_SECRET_KEY || '',
       },
     },
   })
   ```

   **Tip**: At runtime you can override site and secret keys with the `NUXT_TURNSTILE_SECRET_KEY` and `NUXT_PUBLIC_TURNSTILE_SITE_KEY` environment variables.

## Usage

To use Turnstile, you will likely want to:

- Use the `<Turnstile>` component in your app (for example to build a contact form)

> -  Verify the token on your server, when you are processing an API request or a form submission (for example, before sending the email out)

Reviewer: ```suggestion
- Verify the token on your server, when you are processing an API request or a form submission (for example, before sending the email out)
```

> - On your server when you are processing an API request or a form submission, you will want to verify the token (for example, before sending the email out)

Reviewer: ```suggestion
-  Verify the token on your server, when you are processing an API request or a form submission (for example, before sending the email out)
```

</script>
```

### Server

You can either use the a generated validation endpoint, or use the imported helper method:

**Example with endpoint**:

Turn on the generation of the endpoint first:

```js
export default defineNuxtConfig({
  // ...
  turnstile: {
    siteKey: '<your-site-key>',
    addValidateEndpoint: true
  },
})
```

You can now call the endpoint at `/_turnstile/validate` from the client to validate tokens.

> **Example with custom endpoint and helper method**

Reviewer: ```suggestion
**Example with custom endpoint and helper method**:
```
